### DISCUSSION TITLE
How to show more flags?

### DISCUSSION THREAD

**Discussion by ahmedelq** [marlonrichert/zsh-autocomplete#700] [Q&A]
I have flag suggestion for a command, I can see there's an option to show more `(MORE)`. How can I enable it? I searched for all possible keybindings none worked for me. 

![image](https://github.com/marlonrichert/zsh-autocomplete/assets/11134754/ddcbce99-8965-481e-a8d8-3fc2d3d0cdc4)

**Comment by edte** [comment]
press tab

**Reply by ahmedelq** [reply]
`tab` autocompletes with the first suggestion.
